src/provider/cloud-element-templates/components/ElementTemplatesGroup.js

```diff
@@ -0,0 +1,297 @@
+        { label }
+      </div>
+
+      <div class="bio-properties-panel-group-header-buttons">
+        <TemplateGroupButtons element={ element } />
+        { !empty && <SectionToggle open={ open } /> }
+      </div>
+    </div>
+
+    <div class={ classnames(
+      'bio-properties-panel-group-entries',
+      { open: open && !empty }
+    ) }>
+      {
+        entries.map(e => e.component)
+      }
+    </div>
+  </div>;
+}
+
+
+function SectionToggle({ open }) {
+  return <HeaderButton
+    title="Toggle section"
+    class="bio-properties-panel-arrow"
+  >
+    <ArrowIcon class={ open ? 'bio-properties-panel-arrow-down' : 'bio-properties-panel-arrow-right' } />
+  </HeaderButton>;
+}
+
+
+/**
+ *
+ * @param {object} props
+ * @param {object} props.element
+ */
+function TemplateGroupButtons({ element }) {
+  const elementTemplates = useService('elementTemplates');
+
+  const templateState = getTemplateState(elementTemplates, element);
+
+  if (templateState.type === 'NO_TEMPLATE') {
+    return <SelectEntryTemplate element={ element } />;
+  } else if (templateState.type === 'KNOWN_TEMPLATE') {
+    return <AppliedTemplate element={ element } />;
+  } else if (templateState.type === 'UNKNOWN_TEMPLATE') {
+    return <UnknownTemplate element={ element } />;
+  } else if (templateState.type === 'OUTDATED_TEMPLATE') {
+    return <OutdatedTemplate element={ element } templateState={ templateState } />;
+  }
+}
+
+function SelectEntryTemplate({ element }) {
+  const translate = useService('translate');
+  const eventBus = useService('eventBus');
+
+  const selectTemplate = () => eventBus.fire('elementTemplates.select', { element });
+
+  return (
+    <HeaderButton
+      title="SelectEntry a template"
+      class="bio-properties-panel-select-template-button"
+      onClick={ selectTemplate }
+    >
+      <CreateIcon />
+      <span>{ translate('Select') }</span>
+    </HeaderButton>
+  );
+}
+
+function AppliedTemplate({ element }) {
+  const translate = useService('translate'),
+        injector = useService('injector');
+
+  const menuItems = [
+    { entry: translate('Unlink'), action: () => unlinkTemplate(element, injector) },
+    { entry: <RemoveTemplate />, action: () => removeTemplate(element, injector) }
+  ];
+
+  return (
+    <DropdownButton menuItems={ menuItems } class="bio-properties-panel-applied-template-button">
+      <HeaderButton>
+        <span>{ translate('Applied') }</span>
+        <ArrowIcon class="bio-properties-panel-arrow-down" />
+      </HeaderButton>
+    </DropdownButton>
+  );
+}
+
+function RemoveTemplate() {
+  const translate = useService('translate');
+
+  return <span class="bio-properties-panel-remove-template">{ translate('Remove') }</span>;
+}
+
+function UnknownTemplate({ element }) {
+  const translate = useService('translate'),
+        injector = useService('injector');
+
+  const menuItems = [
+    { entry: translate('Unlink'), action: () => unlinkTemplate(element, injector) },
```

Comment:
(minor) Could we arrange in the same order we have for the C7 implementation? (this was changed just recently, and this probably copy pasted prior to that :-) )

c.f. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/545

Reply:
Dammit 😆 Thanks for the notice, I adjusted it!

Another good showcase that we should investigate in making some things, as the `ElementTemplatesGroup`, more agnostic (cf. https://github.com/bpmn-io/bpmn-js-properties-panel/issues/562).